com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -1,14 +1,89 @@
 
-## [0.1.0] - 2020-03-02
-This is the initial experimental Unity MLAPI Package.
-### Changed
-- Rather than performance / convenience RPCs, we now have one performant 'Standard' RPC type
-- RPCs are now deferred and queued to support upcoming features and improve bandwidth usage
-- A preliminary tick system has been added
-- Network variables are tick-based
+## [0.1.0] - 2021-03-23
+
+This is the initial experimental Unity MLAPI Package, v0.1.0.
+
+### New Features
+
+- Refactored a new standard for Remote Procedure Call (RPC) in MLAPI which provides increased performance, significantly reduced boilerplate code, and extensibility for future-proofed code. MLAPI RPC includes `ServerRpc` and `ClientRpc` to execute logic on the server and client-side.
+- Added standarized serialization types, including built-in and custom serialization flows. See [RFC #2](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/blob/master/text/0002-serializable-types.md) for details.
+- `INetworkSerializable` interface replaces `IBitWritable`.
+- Added `NetworkSerializer`..., which is the main aggregator that implements serialization code for built-in supported types and holds `NetworkReader` and `NetworkWriter` instances internally.
+- Added a Network Update Loop infrastructure that aids Netcode systems to update (such as RPC queue and transport) outside of the standard `MonoBehaviour` event cycle. See [RFC #8](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/blob/master/text/0008-network-update-loop.md) and the following details:
+
+  - It uses Unity's [low-level Player Loop API](https://docs.unity3d.com/ScriptReference/LowLevel.PlayerLoop.html) and allows for registering `INetworkUpdateSystem`s with `NetworkUpdate` methods to be executed at specific `NetworkUpdateStage`s, which may also be before or after `MonoBehaviour`-driven game logic execution.
+  - You will typically interact with `NetworkUpdateLoop` for registration and `INetworkUpdateSystem` for implementation.
+  - `NetworkVariable`s are now tick-based using the `NetworkTickSystem`, tracking time through network interactions and syncs.
+
+- Added message batching to handle consecutive RPC requests sent to the same client. `MessageBatcher` sends batches based on requests from the `RpcQueueProcessing`, by batch size threshold or immediately.
+- [GitHub 494](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/494): Added a constraint to allow one `NetworkObject` per `GameObject`, set through the `DisallowMultipleComponent` attribute.
+- Integrated MLAPI with the Unity Profiler for versions 2020.2 and later.
+- A test project is available for building and experimenting with MLAPI features. This project is located in the package `testproject` folder ([GitHub folder](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/tree/release/0.1.0/testproject)). 
```

Comment:
How does this changelog show up for package users?  Reason I ask is, for them they will not have the testproject

Reply:
Hmm can revise pointing them to the test project on github, adding a tweak!